Data-Structures/LinkedList/src/main/java/LinkedList/App.java

```diff
@@ -0,0 +1,23 @@
+/*
```

Comment:
Moving forward you won't really need an App.java or a main method. you'll mostly be working within a folder.